code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -223,13 +229,30 @@
 		sleeperUI(usr)
 		return
 
+	if(href_list["sync"])
+		var/success = 0
+		for(var/obj/machinery/r_n_d/server/S in machines)
```

Comment:
This list is very large, and as far as I'm aware, there's no cooldown on clicking this button in this UI. Might be an easy DDoS target. Maybe you should just have them smush their face against the server since they have access, and are borgs (or add a cooldown).